thirdeye/thirdeye-pinot/src/main/resources/assets/js/lib/heatmap.js

```diff
@@ -53,6 +60,12 @@ function renderD3heatmap(data, tab) {
             root_0.metric = metric;
             root_1.metric = metric;
             root_2.metric = metric;
+            console.log("metric " + metric);
+            if (undefined != invertColorMetrics && invertColorMetrics.indexOf(metric) > -1) { // invert
```

Comment:
proper way to check undefined value is: 

typeof x !== "undefined"
so here: if (typeof x !== "undefined" && invertColorMetrics.indexOf(metric) > -1)